workflow-workflow.md

```diff
@@ -79,6 +91,9 @@ git commit --amend -m "commit message"
 git push --force
 ```
 
+If you use `git commit --amend`, an editor will be launched for you
+to change the commit message.
```

Comment:
Could also mention the `--no-edit` flag if the message should stay the same.
Where is the line between documenting git and documenting workflow? :)